common/workunit/workunit.cpp

```diff
@@ -14218,8 +14221,16 @@ extern WORKUNIT_API void addTimeStamp(IWorkUnit * wu, unsigned wfid, const char
     addTimeStamp(wu, getScopeType(scope), scope, kind, wfid);
 }
 
-static double getCostCpuHour()
+static double getCostCpuHour(const char * subComponentName = nullptr)
```

Comment:
You could remove the default parameter, so you explicitly pass null for the default.